### PR TITLE
Master

### PR DESCRIPTION
**Pull request by flavionegrao**
Added beforeFind readPreferences options

### CODE REVIEW

File: types/parse/index.d.ts [hunk 1]
```diff
@@ -907,22 +907,30 @@ declare namespace Parse {
...
         interface BeforeDeleteResponse extends FunctionResponse { }
         interface BeforeSaveRequest extends TriggerRequest { }
-        interface BeforeFindRequest extends BeforeFindTriggerRequest { }
         interface BeforeSaveResponse extends FunctionResponse {
             success: () => void;
         }
+
+        // Read preference describes how MongoDB driver route read operations to the members of a replica set.
+        enum ReadPreferenceOption {
+            Primary = 'PRIMARY',
```
Comment (DanielRosenwasser) [inline types/parse/index.d.ts]: Does this enum actually exist at runtime? Can you provide documentation for the change?

**Review by DanielRosenwasser — CHANGES_REQUESTED** [review]

**Comment by flavionegrao** [comment]
Apart from being a awesome feature Parse docs are not updated with this at the moment.
But it is implemented since 2.5.0, you can see them implemented reading the unit tests.
https://github.com/parse-community/parse-server/pull/3865/commits <https://github.com/parse-community/parse-server/pull/3865/commits>
https://github.com/parse-community/parse-server/issues/3960 <https://github.com/parse-community/parse-server/issues/3960>
https://github.com/parse-community/parse-server/pull/3865/files#diff-69d2e3afe48f905f42e4bd18eb855beb <https://github.com/parse-community/parse-server/pull/3865/files#diff-69d2e3afe48f905f42e4bd18eb855beb>


Cheers, Flavio

**Comment by flavionegrao** [comment]
Daniel?

**Review by DanielRosenwasser — APPROVED** [review]

**Comment by DanielRosenwasser** [comment]
Thanks for the patience!

**Pull request merged**